lib/iris/tests/unit/aux_factory/test_AuxCoordFactory.py

```diff
@@ -82,35 +91,44 @@ def test_lazy_complex(self):
 
 
 class Test__nd_bounds(tests.IrisTest):
+    def test_numpy_scalar_coord__zero_ndim(self):
+        bounds = np.arange(2)
+        points = np.array(0.5)
```

Comment:
Would you mind swapping the order of points and bounds on the above two lines please?  Nothing functional, just an OCD thing.